cmd/nebula-cert/ca.go

```diff
@@ -220,9 +226,25 @@ func ca(args []string, out io.Writer, errOut io.Writer, pr PasswordReader) error
 		switch *cf.curve {
 		case "25519", "X25519", "Curve25519", "CURVE25519":
 			curve = cert.Curve_CURVE25519
-			pub, rawPriv, err = ed25519.GenerateKey(rand.Reader)
-			if err != nil {
-				return fmt.Errorf("error while generating ed25519 keys: %s", err)
+			if *cf.inKeyPath == "" {
```

Comment:
I'd really like to have this feature, but I think if we're going to add it, we need to support P256 as well.